crates/core/src/table/mod.rs

```diff
@@ -70,7 +72,7 @@ pub(crate) fn get_partition_col_data_types<'a>(
 pub struct DeltaTable {
     /// The state of the table as of the most recent loaded Delta log entry.
     pub state: Option<DeltaTableState>,
-    /// the load options used during load
+    /// options used during load and for data file formatting
```

Comment:
As noted, right now these options only apply during table load and we would be changing them to be more persistant.

Reply:
Yeah that could simplify things, and should mean we don't have to make this serializable to a string I think? The `log_store` is passed through operations using `new_with_state` so we could do something similar with the Parquet options.

Reply:
Yeah I think so. Either way we probably have to update new_with_state no matter what so a new member will likely not create additional work.